Core/Core/Kits/Units.cs

```diff
@@ -10,35 +11,36 @@ public static class Units
   public const string Meters = "m";
   public const string Kilometers = "km";
   public const string Inches = "in";
-  public const string Feet = "ft"; // smelly ones
```

Comment:
🥲 I liked this silly comment hahahaha RIP

Reply:
Perhaps Rider's AI assistant can satisfy your need for humorous dialogue adjacent to our code base.

Reply:
Although, happy to revert this, don't want to step on *too many toes*